### PR TITLE
fix(layout) language selection with locale in URL

### PR DESCRIPTION
**Pull request by abirembaut**
- remove locale parameter from URL when selecting language
- use new [URLSearchparams API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/delete) since IE 11 is no longer supported

Covers [BR-582](https://bonitasoft.atlassian.net/browse/BR-582)

### CODE REVIEW

File: uid-pages/layout-bonita/src/LayoutBonitaV6.json [hunk 1]
```diff
@@ -2594,8 +2594,15 @@
         "if($data.eventList.actions && $data.eventList.actions.length > 0){",
         "    var action=$data.eventList.actions[0];",
         "    if(action===\"locale\"){",
-        "        document.cookie=\"BOS_Locale=\"+$data.selectedLanguage.locale+\";path=/\";;",
-        "        document.location.reload();",
+        "        document.cookie=\"BOS_Locale=\"+$data.selectedLanguage.locale+\";path=/\";",
+        "        try {",
+        "            var url = new URL(document.location.href);",
```
Comment (DumitruCorini) [inline uid-pages/layout-bonita/src/LayoutBonitaV6.json:2599]: maybe a test would be nice for this. Navigate to the url with ?locale and _l and check that it doesn't contain it anymore when changing the language.

Comment (DumitruCorini) [reply uid-pages/layout-bonita/src/LayoutBonitaV6.json:2599]: I can do it if you don't have the time.

Comment (abirembaut) [reply uid-pages/layout-bonita/src/LayoutBonitaV6.json:2599]: I will check if it is possible with cypress

Comment (abirembaut) [reply uid-pages/layout-bonita/src/LayoutBonitaV6.json:2599]: done

**Comment by DumitruCorini** [comment]
LGTM

**Pull request merged**